plugins/parsers/influx/influx_upstream/parser.go

```diff
@@ -21,7 +21,7 @@ const (
 
 var (
 	ErrNoMetric = errors.New("no metric in line")
-	ErrEOF      = errors.New("EOF")
+	ErrEOF      = io.EOF
```

Comment:
Can we please replace this in the code instead of "redefining" EOF?

Reply:
Done.